worker.go

```diff
@@ -166,54 +166,32 @@ func (worker *worker) countThreads() int {
 	return l
 }
 
+func (worker *worker) makeScalingDecision() {
+	autoscaleWorkerThreads(worker)
+}
+
 func (worker *worker) handleRequest(r *http.Request, fc *FrankenPHPContext) {
 	metrics.StartWorkerRequest(fc.scriptFilename)
 
-	// dispatch requests to all worker threads in order
-	worker.threadMutex.RLock()
-	for _, thread := range worker.threads {
-		select {
-		case thread.requestChan <- r:
-			worker.threadMutex.RUnlock()
-			<-fc.done
-			metrics.StopWorkerRequest(worker.fileName, time.Since(fc.startedAt))
-			return
-		default:
-			// thread is busy, continue
-		}
-	}
-	worker.threadMutex.RUnlock()
```

Comment:
I'd like to keep the logic of dispatching requests to workers in order. It was introduced in #1289 and minimizes potential external connections and memory usage when not all threads are fully utilized. In other words, some workers stay 'hot' while higher index workers remain 'cold'. 
It also reduces CPU contention, but that matter less than I initially expected so it's probably fine to remove it for regular threads.